packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8611,6 +8638,29 @@ describe('plugin-meetings', () => {
           );
         });
 
+        it('listens to the brb state changed event', () => {
+          meeting.sendSlotManager.setSourceStateOverride = sinon.stub()
+          meeting.mediaProperties.webrtcMediaConnection = true
```

Comment:
@SomeBody16  How does this line affect the logic for `setUpLocusInfoSelfListener` tests?